lumicks/pylake/population/dwelltime.py

```diff
@@ -38,15 +39,18 @@ def _sample_distributions(self, optimized, iterations):
         """
         n_data = optimized.dwelltimes_sec.size
         self._samples = np.empty((optimized._parameters.size, iterations))
+        min_observation_time, max_observation_time = optimized._observation_limits
         for itr in range(iterations):
             sample = np.random.choice(optimized.dwelltimes_sec, size=n_data, replace=True)
-            result = _kinetic_mle_optimize(
+            result, _ = _exponential_mle_optimize(
                 optimized.n_components,
                 sample,
-                *optimized.observation_limits,
+                min_observation_time,
+                max_observation_time,
```

Comment:
nitpick: but I thought the unpack was just as clean, since you don't use these times anywhere else?